<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-DUERR93" failures="1" tests="4" name="Test" time="30.694" errors="0" timestamp="2024-06-08T23:53:51 IST">
  <testcase classname="TestPack.OrderCreationTest" name="createOrder" time="1.409"/>
  <testcase classname="TestPack.LoginFunctionality" name="LoginTest" time="0.344"/>
  <testcase classname="TestPack.LogoutFunctionality" name="LogoutTest" time="0.586"/>
  <testcase classname="TestPack.InvalidLoginTest" name="testInvalidLogin" time="0.338">
    <failure type="java.lang.AssertionError" message="Error message does not match! expected [Epic sadface: Usrname and password do not match any user in this service] but found [Epic sadface: Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: Error message does not match! expected [Epic sadface: Usrname and password do not match any user in this service] but found [Epic sadface: Username and password do not match any user in this service]
at TestPack.InvalidLoginTest.testInvalidLogin(InvalidLoginTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testInvalidLogin -->
</testsuite> <!-- Test -->
